frontend/src/pages/MetadataByType/SingleMetaDataResult.jsx

```diff
@@ -0,0 +1,216 @@
+import { MetadataURL } from '../../sharedComponents/Metadata/MetadataURL';
+import { ReleaseStateLabel } from '../../sharedComponents/Metadata/ReleaseStateLabel';
+import { alertActions } from '../../state/actions/alert';
+import { FeedbackLabel } from '../../sharedComponents/Metadata/FeedbackLabel';
+
+const SingleMetaDataResultContainer = styled.div`
+  margin: 0.5rem;
+  border: solid 0.2em #F7F9FA;
+  border-radius: 0.5rem;
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+  background-color: #f3f3f3;
+  max-width: 100%;
+  box-shadow: 0 0.1rem 0.25rem #dbdbdb; 
+`;
+const MetaDataContent = styled.div`
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  border-radius: 0.3rem;
+  background-color: white;
+  overflow: hidden;
+  max-width: 100%;
+`;
+const MetaDataDescription = styled.div`
+  flex-grow: 1;
+  padding: 0.5rem;
+  color: #313131;
+  overflow: hidden;
+  max-width: 100%;
+  & > p {
+    word-wrap: break-word;
+    position: relative;
+  }
+`;
+
+const MetaDataMunicipalityLink = styled(Link)`
+  & > p {
+    color: #3e3e3e;
+    font-size: 0.9rem;
+    background-color: #f2f2f2;
+    border-radius: 0.2rem;
+    padding: 0.2rem;
+    display: inline-block;
+  }
+`;
+const MetaDataLink = styled(Link)`
+  padding: 0.4rem;
+  display: inline-block;
+  background-color: #efd8ff;
+  border-radius: 0.2rem;
+  margin: 0.2rem 0;
+  color: rebeccapurple;
+`;
+const MetaDataRating = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  padding: 0.4rem;
+  background-color: #F7F9FA;
+`;
+const CommentsLink = styled(Link)`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  align-content: center;
+
+  & > p {
+    font-size: 0.8rem;
+    max-width: 6rem;
+    text-align: center;
+  }
+  & > img {
+    max-width: 1.4rem;
+  }
+`;
+const Favourite = styled.button`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  align-content: center;
+  border: none;
+  background-color: transparent;
+  padding: 0;
+  cursor: pointer;
+  & > p {
+    font-size: 0.8rem;
+    max-width: 6rem;
+    text-align: center;
+  }
+`;
+const CommentsIcon = styled(Bubbles)`
+  width: 1.4rem;
+  color: #5d5d5d;
+`;
+const StarFullStyled = styled(StarFull)`
+  width: 1.4rem;
+  color: #f9d205;
+`;
+const StarEmptyStyled = styled(StarEmpty)`
+  width: 1.4rem;
+  color: #5d5d5d;
+`;
+
+const SingleMetaDataResult = ({ metadata }) => {
+  const {
+    uuid, formatName, url, description, releaseState, experiencePostGuid, municipalityName,
+  } = metadata;
+
+  // TODO: Update this to use API whenever that exists.
+  const commentsCount = 0;
+  const hasFeedback = experiencePostGuid !== null;
+
+  const dispatch = useDispatch();
+  const [likes, setLikes] = useState(0);
+  const [isLiked, setLiked] = useState(false);
+  const userSelector = useSelector((state) => state.user) || { user: { token: { token: null } } };
+  const { user: token } = userSelector;
```

Comment:
this line assigns value  `userSelector.user` to the name `token`, is this intentional or did you mean `{ user: { token } }`?

Reply:
Also, `{ user: { token: null } }` should be sufficient if you want the same shape as the redux state.